src/App.vue

```diff
@@ -52,20 +52,28 @@
 
 <script lang="ts">
 // Libraries
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Vue, Mixins } from 'vue-property-decorator'
+import { mapState, mapActions } from 'vuex'
```

Comment:
Not sure we need these if we are using the Vuex-class decorators to shortcut the bindings.

Reply:
You are right. We dont need this. Will remove those

Reply:
cool!